mmpose/core/keypoint/__init__.py

```diff
@@ -1,9 +1,10 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+from .codecs import MegviiHeatmap, MSRAHeatmap, UDPHeatmap
```

Comment:
Maybe the codec names should follow the same rule(e.g. the most well-known paper). Names like megvii and msra is a little bit confusing me, and they are different from UDP.

Reply:
What about SBLHeatmap (or SimpleBaselineHeatmap), CPNHeatmap, and UDPHeatmap?

Reply:
These codecs may need further refactoring. We can leave the naming issue to be solved later.